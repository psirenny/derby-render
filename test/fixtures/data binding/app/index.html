<Title:>
  {{foo}}

<Body:>
  <p>{{bar}}</p>
