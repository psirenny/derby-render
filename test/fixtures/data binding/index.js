var fs = require('fs');
var path = require('path');
exports.app = require('./app');
exports.expected = fs.readFileSync(path.join(__dirname, 'expected/index.html'), 'utf8').trim();
