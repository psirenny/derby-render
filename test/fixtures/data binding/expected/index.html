<!DOCTYPE html><meta charset="utf-8"><title>foo</title><body><p>bar</p>
